Header header
uint16 tag_id
uint32 distance
int16 RSS
